pkg/mark/renderer/blockquote.go

```diff
@@ -50,9 +51,10 @@ func (m BlockQuoteLevelMap) Level(node ast.Node) int {
 
 // ClassifyingBlockQuote compares a string against a set of patterns and returns a BlockQuoteType
 func ClassifyingBlockQuote(literal string) BlockQuoteType {
-	infoPattern := regexp.MustCompile(`info|Info|INFO`)
+	infoPattern := regexp.MustCompile(`info|Info|INFO|IMPORTANT|CAUTION`)
```

Comment:
so we should either support all types of upper and lowercase here or (that might be simpler) replace the whole regex with a strings.ToLower(literal) and then just use the lower case in the switch case part later.

Reply:
added to regex `(?i)` for case insensitivity